libqtile/widget/currentlayout.py

```diff
@@ -159,12 +160,7 @@ def _get_layout_names(self):
         """
         Returns the list of lowercased strings for each available layout name.
         """
-        return [
-            layout_class_name.lower()
-            for layout_class, layout_class_name
-            in map(lambda x: (getattr(layout_module, x), x), dir(layout_module))
-            if inspect.isclass(layout_class) and issubclass(layout_class, Layout)
-        ]
+        return [l.__class__.__name__.lower() for l in libqtile.qtile.config.layouts]
```

Comment:
Let's do `self.qtile` rather than `libqtile.qtile` for now.

Reply:
Thank you.